### PR TITLE
Add how to boot Clear via GRUB.

### PR DESCRIPTION
**Pull request by bktan8**
Add dual-boot-linux link to the multi-boot doc.

Signed-off-by: Bun K Tan <bun.k.tan@intel.com>

### CODE REVIEW

File: source/tutorials/multi-boot/dual-boot-linux.rst [hunk 1]
```diff
@@ -166,23 +168,34 @@ Install |CL| with Advanced Installation
 #. Complete the remaining steps of :guilabel:`Required Options` to 
    to install |CL|. Complete any :guilabel:`Advanced Options` as desired.
     
-Ways to boot |CL|
-*****************
+Three Methods to Boot |CL|
+*************(************
```
Comment (mvincerx) [inline source/tutorials/multi-boot/dual-boot-linux.rst]: Looks like you need to make all these asterisks. Minor typo.

File: source/tutorials/multi-boot/dual-boot-linux.rst [hunk 2]
```diff
@@ -166,23 +168,34 @@ Install |CL| with Advanced Installation
...
+
+Although we installed |CL| last, Ubuntu is still the default boot OS.  
+There are three methods to boot |CL|:
 
-Although we installed |CL| last, Ubuntu is still the default boot OS.  There are several ways to boot |CL|:
+#. Make systemd-boot, the boot loader that |CL| uses, the default
```
Comment (mvincerx) [inline source/tutorials/multi-boot/dual-boot-linux.rst]: Please revise this sentence. Something along the lines of:

Make `systemd-boot` the default bootloader for |CL|, and chain-boot GRUB for another distro.

File: source/tutorials/multi-boot/dual-boot-linux.rst [hunk 3]
```diff
@@ -276,6 +289,78 @@ Make systemd-boot Default Bootloader and chain-boot GRUB
...
+
+      .. code-block:: bash
+
+         sudo blkid 
+
+      Example output:
+    
+      .. code-block:: console
+
+         <ADD EXAMPLE HERE>
```
Comment (mvincerx) [inline source/tutorials/multi-boot/dual-boot-linux.rst]: OK. I gather that you'll add an example soon.

File: source/tutorials/multi-boot/dual-boot-linux.rst [hunk 4]
```diff
@@ -276,6 +289,78 @@ Make systemd-boot Default Bootloader and chain-boot GRUB
...
+   allow you select one which OS to boot.
+
+   a. sudoedit /etc/defaults/grub
+
+   #. Set the `GRUB_TIMEOUT` variable to a desire time.
+
+#. Create a menu entry for systemd-boot bootloader.
+
+   a. Identify the UUID for EFI system partition that systemd-boot resides on.
+      The example below shows the UUID for the EFI system on /dev/sda1 is "".
```
Comment (mvincerx) [inline source/tutorials/multi-boot/dual-boot-linux.rst]: ""?

**Comment by mvincerx** [comment]
@bktan8. For clarity, is your intent to close with this PR https://github.com/clearlinux/clear-linux-documentation/issues/908?

File: source/tutorials/multi-boot/dual-boot-linux.rst (hunk 1 above)
Comment (mvincerx) [reply source/tutorials/multi-boot/dual-boot-linux.rst]: Hey @bktan8. Could you please make these changes. 
I tested this. Technically, it builds without errors. Syntax still needs to be fixed.

**Comment by bktan8** [comment]
LGTM

**Comment by mvincerx** [comment]
LGTM.

**Pull request merged**